roles/contrail_master/tasks/main.yaml

```
@@ -0,0 +1,17 @@
+---
+- name: Run kube proxy
+  import_role:
+    name: kube_proxy_and_dns
+  when: inventory_hostname == groups.masters.0
+
+- name: Get contrail nodes and ips
+  include_role:
```

```suggestion
  import_role:
```
to avoid including dynamic roles

---

will resolve this on my next commit